libweb3jsonrpc/JsonHelper.cpp

```diff
@@ -360,8 +360,8 @@ TransactionSkeleton toTransactionSkeleton(Json::Value const& _json)
 
 	if (!_json["from"].empty())
 		ret.from = jsToAddress(_json["from"].asString());
-	if (!_json["to"].empty() && _json["to"].asString() != "0x")
-		ret.to = jsToAddress(_json["to"].asString());
+    if (!_json["to"].empty() && _json["to"].asString() != "0x" && !_json["to"].asString().empty())
```

Comment:
An empty string in `to` field sounds to me like an invalid json for transaction Why do you want to ignore this error and create and invalid `TransactionSkeleton` ?

Reply:
but `"to" : "0x"`  is not considered to be invalid? 

hm. in tests it is empty `to` string that indicates creation.

Reply:
Ah yes, right, sorry empty recipient means contract creation